Ryujinx.Graphics.Gpu/Image/TextureCompatibility.cs

```diff
@@ -147,28 +147,51 @@ public static bool LayoutMatches(TextureInfo lhs, TextureInfo rhs)
         }
 
         /// <summary>
-        /// Checks if the view sizes of a two given texture informations match.
+        /// Obtain the minimum compatibility level of two provided view compatibility results.
+        /// </summary>
+        /// <param name="first">The first compatibility level</param>
+        /// <param name="second">The second compatibility level</param>
+        /// <returns>The minimum compatibility level of two provided view compatibility results</returns>
+        public static TextureViewCompatibility PropagateViewCompatibility(TextureViewCompatibility first, TextureViewCompatibility second)
+        {
+            if (first == TextureViewCompatibility.Incompatible || second == TextureViewCompatibility.Incompatible)
+            {
+                return TextureViewCompatibility.Incompatible;
+            }
+            else if (first == TextureViewCompatibility.CopyOnly || second == TextureViewCompatibility.CopyOnly)
```

Comment:
Assuming this function should "Obtain the minimum compatibility level", should this not check that both textures are CopyOnly compatible, rather than either of them?

```suggestion
            else if (first == TextureViewCompatibility.CopyOnly && second == TextureViewCompatibility.CopyOnly)
```

Reply:
If one texture is CopyOnly and the other is Full, then when considering them both the minimum level of compatibility is copy. It's the same logic as the incompatible check above - it chooses the most restrictive compatibility of the two.

Reply:
:+1: